File: PRAK204/PRAK204-2210817310013-RyanMuhammadIrfan.c
#include <stdio.h>

int main()
{
    float phi, r, t, V, L, K;
    phi = 22.0 / 7.0;

    scanf("%f", &r);
    scanf("%f", &t);

    V = phi * r * r * t;
    L = 2 * phi * r * (r + t);
    K = 2 * phi * r;

    printf("\n");
    printf("Volume = %.2f\n", V);
    printf("Luas = %.2f\n", L);
    printf("Keliling = %.2f\n", K);
}

File: PRAK204/PRAK204-2210817310013-Ryan Muhammad Irfan.py
phi = 22/7

r = int(input(""))
t = int(input(""))

V = phi * r * r * t
L = 2 * phi * r * (r + t)
K = 2 * phi * r

print("")
print("Volume = ", format(V, '.2f'))
print("Luas = ", format(L, '.2f'))
print("Keliling = ", format(K, '.2f'))